.gitignore

```diff
@@ -0,0 +1,36 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
Clicking on = when there is partial operation yet for example 1 + results in "Error", it shouldn't but it should wait for second number
![image](https://github.com/user-attachments/assets/baacec62-417c-4247-af7e-33ff3cd44483)